examples/cloudbuild_repo_connection_gitlab/main.tf

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module "gitlab_connection" {
+  source = "../../modules/cloudbuild_repo_connection"
```

Comment:
For examples please use the registry and version (e.g. https://github.com/terraform-google-modules/terraform-google-bootstrap/blob/master/examples/tf_cloudbuild_builder_simple/main.tf#L18), and the CI will automatically replace with this prior source line when running lint and tests.